virtualbox/machine.go

```diff
@@ -690,13 +690,13 @@ func (m *Machine) Modify() error {
 
 // AddNATPF adds a NAT port forarding rule to the n-th NIC with the given name.
 func (m *Machine) AddNATPF(n int, name string, rule driver.PFRule) error {
-	return vbm("controlvm", m.Name, fmt.Sprintf("natpf%d", n),
+	return vbm("modifyvm", m.Name, fmt.Sprintf("--natpf%d", n),
```

Comment:
Seems to make sense as this is what `docker-machine` does, but what's the effect / reason behind changing this?